project-tracker backend/src/app/routers/timeRecord.ts

```diff
@@ -1,10 +1,13 @@
 import { Router } from 'express';
 import * as timeRecordController from '../controllers/timeRecord';
+import {authorization} from "../../lib/auth";
 
 export const router: Router = Router({ mergeParams: true });
 
-router.get('/', timeRecordController.index);
 router.get('/:timeRecordId', timeRecordController.show);
 router.post('/', timeRecordController.create);
 router.put('/:timeRecordId', timeRecordController.update);
 router.delete('/:timeRecordId', timeRecordController.destroy);
+router.use(authorization)
```

Comment:
ez kifejezetten a timeRecordController.index -nek szól? Mert ha igen, kösd be hozzá közvetlen és nevezd el indexAuthorization-nak.
így: 
router.get('/', timeRecordController.indexAuthorization, timeRecordController.index);

Reply:
egy authorization van az auth.ts-ben, ezt használom mindenhol